code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -28,6 +28,7 @@
 	idle_power_usage = 5
 	layer = BELOW_OBJ_LAYER
 	var/obj/item/reagent_containers/food/snacks/deepfryholder/frying	//What's being fried RIGHT NOW?
+	var/obj/item/clothing/head/mob_holder/frying_mob					//Who's bring fried RIGHT NOW?
```

Comment:
use the correct form of documentation please.

Reply:
There isn't one so I used the same method as the above variable.

Reply:
https://github.com/shiptest-ss13/Shiptest/blob/master/.github/AUTODOC_GUIDE.md